src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.handler.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE_NAME = "src/main/resources/fruits_shop.csv";
+    private static final String OUTPUT_FILE_NAME = "src/main/resources/fruits_report.csv";
+    private static final String BALANCE = "b";
+    private static final String PURCHASE = "p";
+    private static final String SUPPLY = "s";
+    private static final String RETURN = "r";
+
+    public static void main(String[] args) throws RuntimeException {
```

Comment:
```suggestion
    public static void main(String[] args) {
```
should we write that method throws RuntimeException?

Reply:
Of corse shudn't in this case. Sry.